src/Util/Images.php

```diff
@@ -184,12 +184,12 @@ public static function getInfoFromURLCached($url)
 			return $data;
 		}
 
-		$data = DI::cache()->get($url);
+		$data = DI::cache()->get('getInfoFromURL:' . sha1($url));
```

Comment:
For these situations I prefer a `$cachekey` variable that is used both when fetching and writing the cache. This ensures that the same cache key is used.

Reply:
Good point.